core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -215,74 +220,54 @@ var ExplorationEditorTranslationTab = function() {
     await waitFor.visibilityOf(
       languageSelectorElement,
       'Language selector takes too long to appear.');
-    await languageSelectorElement.element(
-      by.cssContainingText('option', language)).click();
+    var languageSelectorElementOption = await languageSelectorElement.element(
+      by.cssContainingText('option', language));
+    await action.click(
+      'Language Selector Element Option ' + language,
+      languageSelectorElementOption);
   };
   var stateNodeLabel = function(nodeElement) {
     return nodeElement.element(by.css('.protractor-test-node-label'));
   };
 
   this.deleteAudioRecord = async function() {
-    await waitFor.elementToBeClickable(
-      deleteRecordButton,
-      'Delete Record button is not clickable');
-    await deleteRecordButton.click();
-    await waitFor.elementToBeClickable(
-      confirmDeleteRecordButton,
-      'The confirm record deletion button is not clickable');
-    await confirmDeleteRecordButton.click();
+    await action.click('Delete Record Button', deleteRecordButton);
+    await action.click(
+      'Confirm Delete Record Button', confirmDeleteRecordButton);
     await waitFor.pageToFullyLoad();
   };
 
   this.uploadAudioRecord = async function(audioPath) {
-    await waitFor.elementToBeClickable(
-      uploadAudioButton,
-      'Audio Record button is not clickable');
-    await uploadAudioButton.click();
+    await action.click('Upload Audio Button', uploadAudioButton);
     absPath = path.resolve(__dirname, audioPath);
     await waitFor.visibilityOf(
       audioUploadInput,
       'Audio upload input field is not visible');
-    await audioUploadInput.sendKeys(absPath);
+    await action.sendKeys('Audio Upload Input', audioUploadInput, absPath);
```

Comment:
There's a nuanced problem here that I just discovered in #11265. We're sending keys here to a file upload element, which cannot be clicked. However, `action.sendKeys` clicks on elements before sending keys to them. In #11265 I changed the `action.sendKeys` function to add a parameter that can disable the clicking behavior. You can wait for that PR to go in, which should be in the next couple days, or copy those changes into this PR

Reply:
This still needs to be addressed